src/utils/components/PolicyForm/PolicyInterface.tsx

```diff
@@ -227,14 +245,31 @@ const PolicyInterface: FC<PolicyInterfaceProps> = ({
             </Flex>
           )}
           {policyInterface?.ipv4 && !policyInterface?.ipv4?.dhcp && (
-            <FormGroup label={t('IPV4 address')} isRequired fieldId={`ipv4-address-${id}`}>
-              <TextInput
-                value={policyInterface?.ipv4?.address?.[0].ip}
-                type="text"
-                id={`ipv4-address-${id}`}
-                onChange={onAddressChange}
-              />
-            </FormGroup>
+            <>
+              <FormGroup label={t('IPV4 address')} isRequired fieldId={`ipv4-address-${id}`}>
+                <TextInput
+                  value={policyInterface?.ipv4?.address?.[0]?.ip}
+                  type="text"
+                  id={`ipv4-address-${id}`}
+                  onChange={onAddressChange}
+                />
+              </FormGroup>
+              <FormGroup label={t('Prefix length')} isRequired fieldId={`prefix-length-${id}`}>
+                <NumberInput
+                  value={policyInterface?.ipv4?.address?.[0]?.['prefix-length']}
+                  id={`prefix-length-${id}`}
+                  onChange={(event) => onPrefixChange(event.currentTarget.valueAsNumber)}
+                  onMinus={() =>
+                    onPrefixChange(policyInterface.ipv4.address[0]['prefix-length'] - 1)
```

Comment:
You are accessing the same data just a few lines above (line 259) and using `?.` there but here not. What is the reason of that? What is the correct form? 
Just a note that if `policyInterface` should always contain `ipv4` and other data, I am fine with not using `?.`. otherwise let's use it. Same for the line 266.

Reply:
because here I'm adding and subtracting 1 so having `undefined + 1` or `undefined - 1` would lead inevitably to an error.
Even if we use `?`. But we have a default value and other checks so I don't think this will happen.